lib/ansible/modules/packaging/language/pip.py

```diff
@@ -125,6 +127,21 @@
     name: bottle
```

Comment:
As stated above I would recommend to have one recommended way to specify the version, i.e change this to `name: bottle==0.11`